.github/workflows/ci.yml

```diff
@@ -19,15 +19,21 @@ jobs:
         uses: styfle/cancel-workflow-action@ce177499ccf9fd2aded3b0426c97e5434c2e8a73
```

Comment:
We can also probably get rid of this, and add that "github concurrency cancel running jobs" stuff. It's kinda funny that we all gravitated toward this one action - I used it before too.